folly/io/async/HHWheelTimer.cpp

```diff
@@ -236,9 +236,7 @@ size_t HHWheelTimer::cancelAll() {
         if (bucket.empty()) {
           continue;
         }
-        for (auto& cb : bucket) {
-          count++;
-        }
+        count += bucket.size();
```

Comment:
That's right. They just uploaded. Funny thing huh?
I'll remove that from this PR so the other useful things are focused.